### DISCUSSION TITLE
How to add CC and BCC emails flow  trigger mails

### DISCUSSION THREAD

**Discussion by suresh20041993** [directus/directus#21901] [Feature Requests]
### Summary

Actually, I trying to send mail to CC and BCC recipients via flow email, but I can see only the To recipient, so how to add CC and BCC 
![Screenshot from 2024-03-19 12-51-47](https://github.com/directus/directus/assets/37900223/b9ff0271-6a82-4ae7-b31f-e9068417caf6)

### Motivation

i was just trying to learn DIRECTUS , send via flows but cc and bcc I can' t add.

### Detailed Design

![Screenshot from 2024-03-19 12-51-47](https://github.com/directus/directus/assets/37900223/8eb767ee-aab7-4821-b374-51b07273627e)

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

![Screenshot from 2024-03-19 12-51-47](https://github.com/directus/directus/assets/37900223/f8498c0e-5347-4b01-84e9-3a3626d0474b)

### Alternatives

![Screenshot from 2024-03-19 12-51-47](https://github.com/directus/directus/assets/37900223/834a241e-604e-4878-916e-6cdef8e66b39)

### Adoption Strategy

![Screenshot from 2024-03-19 12-51-47](https://github.com/directus/directus/assets/37900223/233be437-b794-49cc-8af8-3d7d41d545ec)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
